sdk/storage/test-resources-post.ps1

```diff
@@ -0,0 +1,128 @@
+      <BlobServiceEndpoint>https://$PrimaryAccountName.blob.core.windows.net</BlobServiceEndpoint>
+      <QueueServiceEndpoint>https://$PrimaryAccountName.queue.core.windows.net</QueueServiceEndpoint>
+      <TableServiceEndpoint>https://$PrimaryAccountName.table.core.windows.net</TableServiceEndpoint>
+      <FileServiceEndpoint>https://$PrimaryAccountName.file.core.windows.net</FileServiceEndpoint>
+      <BlobServiceSecondaryEndpoint>https://$PrimaryAccountName-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
+      <QueueServiceSecondaryEndpoint>https://$PrimaryAccountName-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
+      <FileServiceSecondaryEndpoint>https://$PrimaryAccountName-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
+      <TableServiceSecondaryEndpoint>https://$PrimaryAccountName-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
+      <ConnectionString>DefaultEndpointsProtocol=https;AccountName=$PrimaryAccountName;AccountKey=$PrimaryAccountKey;EndpointSuffix=core.windows.net</ConnectionString>
+    </TenantConfiguration>
+    <TenantConfiguration>
+      <TenantName>NamespaceTenant</TenantName>
+      <TenantType>Cloud</TenantType>
+      <AccountName>$DataLakeAccountName</AccountName>
+      <AccountKey>$DataLakeAccountKey</AccountKey>
+      <ActiveDirectoryApplicationId>$AadApplicationId</ActiveDirectoryApplicationId>
+      <ActiveDirectoryApplicationSecret>$AadApplicationSecret</ActiveDirectoryApplicationSecret>
+      <ActiveDirectoryTenantId>$AadTenantId</ActiveDirectoryTenantId>
+      <ActiveDirectoryAuthEndpoint>https://login.microsoftonline.com/</ActiveDirectoryAuthEndpoint>
+      <BlobServiceEndpoint>https://$DataLakeAccountName.blob.core.windows.net</BlobServiceEndpoint>
+      <QueueServiceEndpoint>https://$DataLakeAccountName.queue.core.windows.net</QueueServiceEndpoint>
+      <TableServiceEndpoint>https://$DataLakeAccountName.table.core.windows.net</TableServiceEndpoint>
+      <FileServiceEndpoint>https://$DataLakeAccountName.file.core.windows.net</FileServiceEndpoint>
+      <BlobServiceSecondaryEndpoint>https://$DataLakeAccountName-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
+      <QueueServiceSecondaryEndpoint>https://$DataLakeAccountName-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
+      <FileServiceSecondaryEndpoint>https://$DataLakeAccountName-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
+      <TableServiceSecondaryEndpoint>https://$DataLakeAccountName-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
+    </TenantConfiguration>
+  </TenantConfigurations>
+</TestConfigurations>"
+
+# Construct the test configuration path to use based on the devops build variable for artifact staging directory
+$TestConfigurationPath = Join-Path -Path $env:BUILD_ARTIFACTSTAGINGDIRECTORY -ChildPath 'TestConfiguration.xml'

```

Comment:
I think there are enough open questions for getting this to work on local machines that it probably can be tackled separately. What do you think?

Reply:
Nothing is published from that directory unless you tell it to. These standard directories in DevOps are more like "recommendations" for places to put things.

Reply:
I don't know that everyone agrees we want to move away from the config file entirely, but we can tackle running locally in a separate PR.  I just want to make sure all of those issues get addressed by the folks driving this work across the team.

Reply:
Ah, I see what you're getting at. Yes, you should probably use something `$env:TEMP`, @JoshLove-msft. A common approach if you need a directory for multiple files is something like `Join-Path $env:TEMP ([Guid]::NewGuid().ToString('n'))`.

Reply:
We just have one file that we are creating. I haven't actually changed the file location from what it was before. Are you saying we should just not use the artifact staging directory so that we can more easily persist locally and also to prevent this from ever being published?

Reply:
It won't get published unless something is blindly collecting and publishing all files in that directory, which is unlikely. Nothing does it automatically.

But, yes, I think if this is to be run by other users outside of DevOps, it should use a directory standard to everyone. If not an issue, then I think this directory is fine.